OKTA-449821: Splunk cloud log streaming dev docs (#2991)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

<ApiOperation method="get" url="/api/v1/meta/schemas/logStream/${typeId}" />

> Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which can be the `aws_eventbridge`, literal to retrieve the AWS EventBridge type schema or the `splunk_cloud_logstreaming`, literal to retrieve the Splunk Cloud type schema.

Reviewer: ```suggestion
Fetches the schema for a Log Stream type. The `${typeId}` element in the URL specifies the Log Stream type, which is either `aws_eventbridge` or `splunk_cloud_logstreaming`. Use the `aws_eventbridge` literal to retrieve the AWS EventBridge type schema, and use the `splunk_cloud_logstreaming` literal retrieve the Splunk Cloud type schema.
```

        "name": "Name can't exceed 100 characters."
      }
    }
  },
  {
    "$schema": "https://json-schema.org/draft/2020-12/schema",

>     "$id": "http://${yourOktaDomain}/api/v1/meta/schemas/logStream/splunk_cloud_logstreaming",

Reviewer 2: ```suggestion
    "$id": "http://{yourOktaDomain}/api/v1/meta/schemas/logStream/splunk_cloud_logstreaming",
```
